src/main.tsx

```diff
@@ -36,9 +37,14 @@ export default class AnnotatorPlugin extends Plugin implements IHasAnnotatorSett
     // @ts-ignore: initialized by loadSettings() in onloadImpl()
     settings: AnnotatorSettings;
     views: Set<AnnotatorView> = new Set();
+    observer: MutationObserver;
+    targetClassNameSet: Set<string> = new Set();
 
     public pdfAnnotatorFileModes: { [file: string]: string } = {};
     private _loaded = false;
+    private eventRefs: EventRef[] = [];
```

Comment:
Use an array to store the events of Obsidian, which is convenient for closing events when closing the software